packages/dashboard-core-plugins/src/panels/IrisGridPanel.tsx

```diff
@@ -423,6 +423,11 @@ export class IrisGridPanel extends PureComponent<
         return null;
       }
 
+      // TODO: Find a better way to handle deprecated panel prop
```

Comment:
For a TODO like this, we should reference a ticket number and have a ticket created. We can clone the current ticket, or just use the same ticket since this is kind of an interim solution. We want something better.

```suggestion
      // TODO #2096: Find a better way to handle deprecated panel prop
```